Doc/library/time.rst

```diff
@@ -708,6 +626,101 @@ The module defines the following functions and data items:
       ('EET', 'EEST')
 
 
+Clock ID Constants
+------------------
+
```

Comment:
Good point. How about removing clock_settime() from the paragraph below and add a note to CLOCK_REALTIME documentation that says this constant can only be passed to time.clock_settime()?